docs/guides/chip_tool_guide.md

```diff
@@ -264,8 +273,8 @@ In this command:
 
 ##### Commissioning with long discriminator
 
-Use the following command pattern to discover devices with a long discriminator
-and try to pair with the first discovered one using the provided setup code.
+To discover devices with a long discriminator and try to pair with the first
+discovered one using the provided setup code, use the following command pattern:
```

Comment:
Again, nowadays this tries all matching things. Again, followup?

Reply:
Will fix in a separate PR.

Reply:
Yep, sounds good. Just drop a link to the issue tracking it here and resolve the conversation.